pkg/k8s/apis/cilium.io/register.go

```diff
@@ -15,5 +15,5 @@ const (
 	//
 	// Maintainers: Run ./Documentation/check-crd-compat-table.sh for each release
 	// Developers: Bump patch for each change in the CRD schema.
-	CustomResourceDefinitionSchemaVersion = "1.26.12"
+	CustomResourceDefinitionSchemaVersion = "1.26.13"
```

Comment:
Ah looks like we've missed updating this. Cilium v1.15 will use 1.28.x and Cilium v1.16 (main) will use 1.29.x.
```suggestion
	CustomResourceDefinitionSchemaVersion = "1.29.0"
```

Reply:
(I already bumped the other stable branches last year, only `main` is out of sync)

Reply:
Saw this when rebasing to main :-)